### PR TITLE
micro_ros_platformio works on Raspberry Pi Pico

### PR DESCRIPTION
**Pull request by delihus**
Presented example works properly on RPi Pico

platformio.ini:
``` bash
[env:pico]
platform = raspberrypi
board = pico
framework = arduino
board_microros_transport = serial
lib_deps =
    https://github.com/delihus/micro_ros_platformio
```

logs
``` bash
[1658131559.396250] debug    | DataWriter.cpp     | write                    | [** <<DDS>> **]        | client_key: 0x00000000, len: 4, data: 
0000: 74 00 00 00
[1658131559.396320] debug    | SerialAgentLinux.cpp | send_message             | [** <<SER>> **]        | client_key: 0x653160D0, len: 13, data: 
0000: 81 00 00 00 0A 01 05 00 79 00 00 00 80
[1658131560.397140] debug    | SerialAgentLinux.cpp | recv_message             | [==>> SER <<==]        | client_key: 0x653160D0, len: 16, data: 
0000: 81 80 79 00 07 01 08 00 00 83 00 05 75 00 00 00
[1658131560.397504] debug    | DataWriter.cpp     | write                    | [** <<DDS>> **]        | client_key: 0x00000000, len: 4, data: 
0000: 75 00 00 00
[1658131560.397600] debug    | SerialAgentLinux.cpp | send_message             | [** <<SER>> **]        | client_key: 0x653160D0, len: 13, data: 
0000: 81 00 00 00 0A 01 05 00 7A 00 00 00 80
[1658131561.398154] debug    | SerialAgentLinux.cpp | recv_message             | [==>> SER <<==]        | client_key: 0x653160D0, len: 16, data: 
0000: 81 80 7A 00 07 01 08 00 00 84 00 05 76 00 00 00
[1658131561.398494] debug    | DataWriter.cpp     | write                    | [** <<DDS>> **]        | client_key: 0x00000000, len: 4, data: 
0000: 76 00 00 00
[1658131561.398629] debug    | SerialAgentLinux.cpp | send_message             | [** <<SER>> **]        | client_key: 0x653160D0, len: 13, data: 
0000: 81 00 00 00 0A 01 05 00 7B 00 00 00 80
[1658131562.399237] debug    | SerialAgentLinux.cpp | recv_message             | [==>> SER <<==]        | client_key: 0x653160D0, len: 16, data: 
0000: 81 80 7B 00 07 01 08 00 00 85 00 05 77 00 00 00
[1658131562.399634] debug    | DataWriter.cpp     | write                    | [** <<DDS>> **]        | client_key: 0x00000000, len: 4, data: 
0000: 77 00 00 00
[1658131562.399842] debug    | SerialAgentLinux.cpp | send_message             | [** <<SER>> **]        | client_key: 0x653160D0, len: 13, data: 
0000: 81 00 00 00 0A 01 05 00 7C 00 00 00 80
```

### CODE REVIEW

**Comment by Acuadros95** [comment]
Lookd good!

Could you add this board/transport to the CI ([here](https://github.com/micro-ROS/micro_ros_platformio/blob/main/.github/workflows/ci.yml#L17) and [here](https://github.com/micro-ROS/micro_ros_platformio/blob/main/ci/platformio.ini#L31))?

**Comment by delihus** [comment]
Added

File: README.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ Supported boards are:
 | `olimex_e407`                                | `ststm32`     | `arduino`   | `serial`                         | `colcon.meta`            |
 | `esp32dev`                                   | `espressif32` | `arduino`   | `serial` <br/> `wifi`            | `colcon.meta`            |
 | `nanorp2040connect`                          | `raspberrypi` | `arduino`   | `serial` <br/> `wifi_nina`       | `colcon_verylowmem.meta` |
+| `pico`                                       | `raspberrypi` | `arduino`   | `serial`                         | `colcon.mata`|
```
Comment (Acuadros95) [inline README.md:42]:
```suggestion
| `pico`                                       | `raspberrypi` | `arduino`   | `serial`                         | `colcon.meta`|
```

**Review by Acuadros95 — APPROVED** [review]

File: ci/platformio.ini [hunk 2]
```diff
@@ -118,6 +118,14 @@ lib_ldf_mode = chain+
 board_microros_transport = serial
 lib_deps =
     ../
+    
+[env:pico]
+platform = raspberrypi
+board = pico
+framework = arduino
```
Comment (Acuadros95) [inline ci/platformio.ini:125]:
```suggestion
framework = arduino
lib_ldf_mode = chain+
```

Also add it to [Readme Known Issues/Limitations](https://github.com/micro-ROS/micro_ros_platformio#known-issueslimitations)

**Review by Acuadros95 — APPROVED** [review]

**Pull request merged**